test/ad/distributions.jl

```diff
@@ -408,13 +408,10 @@
             # PoissonBinomial fails with Zygote
             # Matrix case does not work with Skellam:
             # https://github.com/TuringLang/DistributionsAD.jl/pull/172#issuecomment-853721493
-            filldist_broken = if D <: Skellam
-                ((d.broken..., :Zygote, :ReverseDiff), (d.broken..., :Zygote, :ReverseDiff))
-            elseif D <: PoissonBinomial
+            filldist_broken = if D <: PoissonBinomial
                 ((d.broken..., :Zygote), (d.broken..., :Zygote))
             elseif D <: Chernoff
-                # Zygote is not broken with `filldist`
-                ((), ())
+                ((d.broken..., :Zygote), (d.broken..., :Zygote))
```

Comment:
Seems it is not broken (as before), see https://github.com/TuringLang/DistributionsAD.jl/runs/7724467303?check_suite_focus=true#step:6:420:
```suggestion
                # Zygote is not broken with `filldist`
                ((), ())
```